src/mixins/date-mixin.ts

```diff
@@ -1,252 +1,145 @@
-import { Component } from 'vue-property-decorator'
-import { isDate } from 'lodash'
 import { CommonMixin } from '@/mixins/'
+import Vue from 'vue'
+import DateUtilities from '@/services/date-utilities'
+import { Component } from 'vue-property-decorator'
 
 /**
  * Mixin that provides some useful date utilities.
  */
-@Component({})
-export default class DateMixin extends CommonMixin {
+@Component({
+  mixins: [
+    CommonMixin
+  ]
+})
+export default class DateMixin extends Vue {
   /**
-   * Fetches and returns the web server's current date (in UTC).
+   * DEPRECATED Fetches and returns the web server's current date (in UTC).
```

Comment:
The date-mixin now becomes a wrapper for the date-utilities service.  As a service, I can use these functions in the store getters whereas as a mixin, I cannot.

Reply:
This is cool. I like it!